cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java

```diff
@@ -238,6 +246,12 @@ public String getFormat() {
             "Modeling authority set",
             "powsybl.org");
 
+    private static final Parameter MODEL_DESCRIPTION_PARAMETER = new Parameter(
+            MODEL_DESCRIPTION,
+            ParameterType.STRING,
+            "Model description",
+            null);
+
     private static final List<Parameter> STATIC_PARAMETERS = List.of(
```

Comment:
the new parameter `MODEL_DESCRIPTION_PARAMETER` should be added to the list of parameters, `STATIC_PARAMETERS`.